src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -21,9 +22,9 @@
     private Game game;
     private final Player host;
     private final Map<String, Player> playerMap;
-    private final String invitationCode;
-    private final static String HANNIBAL_URL = "https://sopra-fs22-group-16-client.herokuapp.com?=";
-    private final static String QR_API_URL = "https://api.qrserver.com/v1/create-qr-code";
+    private String invitationCode;
+    private byte[] qrCode;
+    private final String HANNIBAL_URL = "https://sopra-fs22-group-16-client.herokuapp.com?=";
```

Comment:
why not static?
"static makes it uniform value across all the class instances. It ensures that you don't waste memory creating many of the same thing if it will be the same value for all the objects."
we will have multiple lobbies.

Reply:
Was a merge mistake, will change it back

Reply:
How will this work for local testing?

Reply:
In my test I made a spy of the lobby and mocked the getQRCode method.

Reply:
Do we want to test the getQRCode method? (In realty yes) but then every time we run full tests we make a call to the qrCode service. Other possibility would be to mock the restTemplate.getForObject(...).

Reply:
I would do so until we're told not to do.
The call to the qrCode service is cheap for us and we have to check whether our call still works or not.
Maybe they change the endpoint or whatever.

Reply:
makes sense